editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -6039,6 +6039,9 @@ void CanvasItemEditorViewport::_create_preview(const Vector<String> &files) cons
 					Node *instance = scene->instance();
 					if (instance) {
 						preview_node->add_child(instance);
+						preview_node->set_pause_mode(PAUSE_MODE_STOP);
```

Comment:
After the pause rework, you no longer need to pause the tree, just set `process_mode` of the `preview_node` to `PROCESS_NODE_DISABLED`. It will only stop the preview node, making the solution a one-liner.